### PR TITLE
Unique labels for feature flag buttons

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

DAC December 2020 audit spotted an issue with repeated labels for these buttons.

## Changes proposed in this pull request

Adds visually hidden text describing the feature that can be activated/deactivated.

## Link to Trello card

https://trello.com/c/QitTcgV3/2764-dac-quick-wins

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/support_interface/settings/feature_flags.html.erb [hunk 1]
```diff
@@ -28,10 +28,8 @@
   ].compact
   ) do %>
     <%= render SummaryCardHeaderComponent.new(title: feature_name.humanize, heading_level: 3) do %>
-      <% if FeatureFlag.active?(feature_name) %>
-        <%= button_to 'Deactivate', support_interface_deactivate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' %>
-      <% else %>
-        <%= button_to 'Activate', support_interface_activate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' %>
+      <%= button_to support_interface_deactivate_feature_flag_path(feature_name), class: 'govuk-button app-button--tertiary govuk-!-margin-bottom-0' do %>
+        <%= FeatureFlag.active?(feature_name) ? 'Deactivate' : 'Activate' %><span class="govuk-visually-hidden"> ‘<%= feature_name.humanize %>’ feature</span>
```
Comment (stevehook) [inline app/views/support_interface/settings/feature_flags.html.erb:32]: Extra hidden label looks good but I think this is always going to link to the `support_interface_deactivate_feature_flag_path` and not the `support_interface_activate_feature_flag_path` that logic needs to be wrapped in a ternary conditional or reverted back to the long-hand `if-else-end`.

**Review by stevehook — COMMENTED** [review]
1 question inline

File: app/views/support_interface/settings/feature_flags.html.erb (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/support_interface/settings/feature_flags.html.erb:32]: Argh! Good spot!

**Review by stevehook — APPROVED** [review]
:+1: Works for me on the review app

**Pull request merged**